src/implementations/twig/components/page-header/page-header.html.twig

```diff
@@ -65,6 +70,17 @@
   {% endfor %}
 {% endif %}
 
+{# Backward compatibility #}
+
+{% if _picture.img is empty and _thumbnail.src is not empty %}
+  {% set _picture = {
+    'img': {
+      'src': _thumbnail.src|default(''),
```

Comment:
you check for the emptyness of the thumbnail.src already why setting its default here..?